packages/book-updater/src/book_updater/processing/post_processors.py

```
@@ -762,3 +762,73 @@ def _find_missing_tags(self, original_block: str, processed_block: str) -> List[
+    "RemoveBlankLinesInListProcessor",
+    "RemoveXmlTagsProcessor",
+    "RemoveMarkdownBlocksProcessor",
+    "RemoveTrailingWhitespaceProcessor",
+    "RevertRemovedBlockLines",
+    "NoNewHeadersPostProcessor",
+    "OrderQuoteAnnotationProcessor",
+    "EmptySectionError",
+    "ValidateNonEmptySectionProcessor",
+    "PreserveFStringTagsProcessor",
+    "InlineQuoteProcessor",
+]
```

Ruff (0.14.11)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)